webapp/channels/src/components/advanced_text_editor/advanced_text_editor.tsx

```diff
@@ -418,14 +418,19 @@ const AdvanceTextEditor = ({
                         role='application'
                         id='advancedTextEditorCell'
                         data-a11y-sort-order='2'
-                        aria-label={Utils.localizeMessage(
-                            'channelView.login.successfull',
-                            'Login Successfull',
-                        ) + ' ' + ariaLabelMessageInput}
+                        aria-describedby={`advancedTextEditorLabel-${location}`}
                         tabIndex={-1}
                         className='AdvancedTextEditor__cell a11y__region'
                     >
                         {labels}
+                        <label
+                            id={`advancedTextEditorLabel-${location}`}
+                            className='sr-only'
+                        >
+                            {Utils.localizeMessage(
+                                'channelView.login.successfull',
+                                'Login Successfull',
+                            ) + ' ' + ariaLabelMessageInput}</label>
```

Comment:
please use react intl methods here

Reply:
I don't think it's a problem specific to the desktop application. When I tested it on the master branch in Safari, it still couldn't detect Grammarly Mattermost as in the screenshots. 

<img width="1470" alt="Ekran Resmi 2023-10-24 23 18 09" src="https://github.com/mattermost/mattermost/assets/62356575/b6e4aa45-20a5-428a-a52f-efa916e94331">
<img width="1470" alt="Ekran Resmi 2023-10-24 23 18 13" src="https://github.com/mattermost/mattermost/assets/62356575/b2eea97f-ad0b-4020-8e1d-551512fa8c7d">
<img width="1470" alt="Ekran Resmi 2023-10-24 23 17 13" src="https://github.com/mattermost/mattermost/assets/62356575/ac721cf8-a44f-4700-ab4b-7388158264ec">
<img width="1465" alt="Ekran Resmi 2023-10-24 23 17 05" src="https://github.com/mattermost/mattermost/assets/62356575/bb64a640-7c37-46cb-9e87-c4cd5442def5">